tests/zndraw/test_socketio_rooms.py

```
@@ -716,7 +760,7 @@ async def test_rest_rejects_updating_other_users_session_camera(
     await sio_a.connect(server_auth, auth={"token": token_a})
     tsio_a = wrap(sio_a)
     result_a = await tsio_a.call(
-        RoomJoin(owner_id=oid, room_name=room_name, client_type="frontend"),
+        RoomJoin(owner=display_name, room_name=room_name, client_type="frontend"),
```

Ruff (0.15.14)

[error] Undefined name `display_name`

(F821)